runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/graph/ImmutableExecutableStageTest.java

```diff
@@ -58,11 +60,13 @@ public void ofFullComponentsOnlyHasStagePTransforms() throws Exception {
                             .setDoFn(RunnerApi.SdkFunctionSpec.newBuilder().setEnvironmentId("foo"))
                             .putSideInputs("side_input", RunnerApi.SideInput.getDefaultInstance())
                             .putStateSpecs("user_state", RunnerApi.StateSpec.getDefaultInstance())
+                            .putTimerSpecs("timer", RunnerApi.TimerSpec.getDefaultInstance())
                             .build()
                             .toByteString()))
             .build();
     PCollection input = PCollection.newBuilder().setUniqueName("input.out").build();
     PCollection sideInput = PCollection.newBuilder().setUniqueName("sideInput.in").build();
+    PCollection timer = PCollection.newBuilder().setUniqueName("timer.out").build();
```

Comment:
It is confusing to reference the single timer collection as "timer.out" when it is used for both input and output.